process-test-results/allure-configuration/categories.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "name": "Ignored tests",
+    "matchedStatuses": [
+      "skipped"
+    ]
+  },
+  {
+    "name": "Port Mishandling",
+    "matchedStatuses": [
+      "broken",
+      "failed"
+    ],
+    "messageRegex": ".*port conflict.*"
+  },
+  {
+    "name": "Product defects",
+    "matchedStatuses": [
+      "failed"
+    ]
+  },
+  {
+    "name": "Test defects",
+    "matchedStatuses": [
+      "broken"
```

Comment:
does "broken" here mean the test is "flaky"?

Reply:
nope, it's a failure.   I've not seen flaky data that didn't result in a full test failure yet (all 3 executions of the test resulted in errors).   Testing out flaky behavior will likely require causing a test to flake intentionally once this is in.